library.go

```diff
@@ -11,10 +11,20 @@ import (
 // UserHasTracks checks if one or more tracks are saved to the current user's
 // "Your Music" library.
 func (c *Client) UserHasTracks(ctx context.Context, ids ...ID) ([]bool, error) {
+	return c.libraryContains(ctx, "tracks", ids...)
+}
+
+// UserHasAlbums checks if one or more albums are saved to the current user's
+// "Your Albums" library.
+func (c *Client) UserHasAlbums(ctx context.Context, ids ...ID) ([]bool, error) {
+	return c.libraryContains(ctx, "albums", ids...)
+}
+
+func (c *Client) libraryContains(ctx context.Context, typ string, ids ...ID) ([]bool, error) {
 	if l := len(ids); l == 0 || l > 50 {
-		return nil, errors.New("spotify: UserHasTracks supports 1 to 50 IDs per call")
+		return nil, errors.New("spotify: UserHasAlbums supports 1 to 50 IDs per call")
```

Comment:
Ah my bad. This error is now wrong.
```suggestion
		return nil, errors.New("spotify: only supports 1 to 50 IDs per call")
```